docs/pages/apis/rest.fr.mdx

```diff
@@ -0,0 +1,123 @@
+# REST API
+
+Notre API RESTful fourni une façon simple d'intéragir avec la base de données de "Potter DB", en adhérant aux spécifications JSON:API et OAS. Dans cette section, nous aborderons les principaux aspects de l'utilisation de l'API REST, ce qui inclue les endpoints disponibles et la manière de formatter vos requêtes et réponses.
+
+## Endpoints disponibles
+
+Pour accéder aux données avec l'API REST vous devez faire une requête en `GET` sur l'un des endpoints suivants :
+
+<details>
+<summary>Dérouler pour voir tous les endpoints</summary>
+
+**Livres**
+- `/v1/books`: Accéder à la liste des livres.
+- `/v1/books/{id}`: Accéder à un livre spécifique via son ID.
+- `/v1/books/{id_du_livre}/chapters`: Accéder à la liste des chapitres d'un livre en particulier.
+- `/v1/books/{id_du_livre}/chapters/{id}`: Accéder à un chapitre spécifique d'un livre particulier, via l'ID du chapitre.
+
+**Personnages**
+- `/v1/characters`: Accéder à la liste des personnages.
+- `/v1/characters/{id}`: Accéder à un personnage en particulier via son ID.
+
+**Films**
+- `/v1/movies`: Accéder à la liste des films.
+- `/v1/movies/{id}`: Accéder à un film en particulier via son ID.
+
+**Potions**
+- `/v1/potions`: Accéder à la liste des potions.
+- `/v1/potions/{id}`: Accéder à une potion en particulier via son ID.
+
+**Sorts**
+- `/v1/spells`: Accéder à la liste des sorts.
+- `/v1/spells/{id}`: Accéder à un sort en particulier via son ID.
+
+Les IDs doivent être fournies au format UUID ou en slug
+
+</details>
+
+## Spécifications OpenAPI (OAS)
+
+Pour faciliter l’intégration, nous fournissons un document de [spécification OpenAPI](https://openapis.org) qui décrit les endpoints, les modèles de données et les schémas de requête/réponse de l’API.
+L’API est conforme à la version 3.0.3 de la spécification OAS.
+Vous trouverez notre document OAS actuel [ici](https://api.potterdb.com/v1/openapi.json).
+
+## Format JSON:API
+
+Notre API REST adhère à la [spécification JSON:API](https://jsonapi.org/), qui fournit un moyen cohérent de structurer les requêtes et les réponses de l’API. Voici un aperçu des principales caractéristiques de JSON:API dans le contexte de notre API :
+
+- **Objets** : Chaque ressource est représentée comme un objet avec un type, un `id` et des attributs (`attributes`).
+- **Relations** : Les ressources peuvent être liées à d’autres ressources, par exemple une ressource « livre » (`book`) est liée aux ressources « chapitre » (`chapter`) et vice versa.
+- **Pagination** : Lorsqu’une réponse contient un grand nombre de ressources, la réponse sera paginée. Voir [Pagination](#pagination) pour plus d’informations.
+- **Erreurs** : Les erreurs sont retournées avec des structures normalisées, ce qui facilite la gestion et le dépannage des problèmes.
```

Comment:
```suggestion
- **Objets**: Chaque ressource est représentée comme un objet avec un type, un `id` et des attributs (`attributes`).
- **Relations**: Les ressources peuvent être liées à d’autres ressources, par exemple une ressource « livre » (`book`) est liée aux ressources « chapitre » (`chapter`) et vice versa.
- **Pagination**: Lorsqu’une réponse contient un grand nombre de ressources, la réponse sera paginée. Voir [Pagination](#pagination) pour plus d’informations.
- **Erreurs**: Les erreurs sont retournées avec des structures normalisées, ce qui facilite la gestion et le dépannage des problèmes.
```
In addition I am not sure whether we should have it like « chapitre » or «chapitre» (without whitespaces). What do you think?